antora-ui-camel/src/css/frontpage.css

```diff
@@ -217,6 +218,11 @@ section.frontpage h1 {
   width: 27rem;
 }
 
+.evaIcons {
```

Comment:
I'd refrain from using `evaIcons`, these are icons that depict functionality so better name for them would be `functionality-icon` or similar. Read [this](https://www.w3.org/QA/Tips/goodclassnames).